### PR TITLE
gradient accumulation

### PR DESCRIPTION
**Pull request by sblackburn86**
Adding a kwarg to allow for gradient accumulation.
In the normal MACE and the MLP score network, there is no batchnorm, so doing gradient accumulation is easy.
Here, just passing an argument to pytorch-lightning trainer does the trick.

TBD what happens with DiffusionMACE with the o3.batchnorm...

### CODE REVIEW

**Review by rousseab — CHANGES_REQUESTED** [review]
The PR only shows changes in the config files. The new argument is not passed to the Trainer... Maybe changes in `train_diffusion.py` were not committed?

**Review by rousseab — APPROVED** [review]
LGTM!
